### DISCUSSION TITLE
Alignment of result column

### DISCUSSION THREAD

**Discussion by hgleich** [sqlitebrowser/sqlitebrowser#3506] [Q&A]
Is there a chance to right-align a particular result column of an sql query? The sqlite3 command line shell accepts negative values in the .width dot command.
Any help is welcome.

**Comment by chrisjlocke** [comment]
![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/7872218/4c7434ab-50bc-4a2d-aa90-7aeb176f5a8b)

Not in the SQL query window.  Figures are right aligned, while text is left aligned.

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/7872218/ec48f908-958d-4573-86d0-235950a0ef0e)

However, the 'Browse Data' tab allows conditional formatting, which includes text alignment.  You can therefore set a condition on a column as needed.  So field3 is right-aligned, field4 is left-aligned (default).

**Reply by hgleich** [reply]
Even while it is not what I hoped to hear: thank you!
